develop.py

```diff
@@ -15,7 +15,7 @@
 import shutil
 import subprocess
 import sys
-from distutils.sysconfig import get_config_var
+from setuptools._distutils.sysconfig import get_config_var
```

Comment:
This doesn't look correct, the new version of the code uses a private submodule of setuptools